lxd/devlxd.go

```diff
@@ -70,12 +70,30 @@ func devlxdConfigGetHandler(d *Daemon, c instance.Instance, w http.ResponseWrite
 	}
 
 	filtered := []string{}
+	hasSSHKeys := false
+	hasCustomConfig := false
 	for k := range c.ExpandedConfig() {
-		if strings.HasPrefix(k, "user.") || strings.HasPrefix(k, "cloud-init.") {
+		// cloud-init.ssh-keys keys should not be retireved by cloud-init directly but instead should be merged into
```

Comment:
retireved

Reply:
I think a comment here explaining why we are merging the SSH keys into both vendor and user data would be useful in the future, as it seems counter intuitive.